article/_posts/2021-01-14-MedAL.md

```diff
@@ -0,0 +1,113 @@
+---
+layout: review
+title: "MedAL: Accurate and Robust Deep Active
+Learning for Medical Image Analysis"
+tags: active-learning deep-learning medical classification
+author: "Thierry Judge"
+cite:
+    authors: "Asim Smailagic, Hae Young Noh, Aure´lio Campilho, Pedro Costa, Devesh Walawalkar, Kartik Khandelwal,
+Mostafa Mirshekari, Jonathon Fagert, Adrian Galdran, Susu Xu"
+    title:   "MedAL: Accurate and Robust Deep Active
+Learning for Medical Image Analysis"
+    venue:   "ICMLA 2018"
+pdf: "https://arxiv.org/ftp/arxiv/papers/1809/1809.09287.pdf"
+---
+
+# Introduction
+
+The authors introduce a novel method for active learning that aims to select more representative samples that standard active learning heuristics. 
+
+# Summary of Contributions
+
+* Novel AL sampling method that maximizes the representativeness of samples selected for labeling.
+* Better initialization by selecting the initial pool with standard feature extraction and description methods
+
+# Existing Methods 
+
+Consider an empty training set $$D_{train}$$ and an unlabeled dataset  $$D_{oracle}$$
+
+Standard Active learning loop
+
+1. Label $n$ samples from  $$D_{oracle}$$ and add them to $$D_{train}$$ .
+2. Train model on $$D_{train}$$ .
+3. Evaluate the model's uncertainty on $$D_{oracle}$$ .
+4. Label $$n$$ most uncertain samples and add them to $$D_{train}$$ .
+5. Repeat steps 2 to 4.
+
+Many heuristics can be used to evaluate uncertainty. 
+
+# Method
+
+## Sampling method
+
+In this work, the authors use rich data descriptions $$f(x) \in R^n$$ to select the most representative sample. This is done by evaluating the average distance with all the samples in $$D_{train}$$. 
+
+
+$$
+S(x) = \frac{1}{N}\sum_{i=1}^N d(f(x_i), f(x))
+$$
+
+
+where $$x_i \in D_{train}$$.  
+
+The rich data descriptor $$f(x)$$ used is a deep representation obtained from a layer in CNN trained for classification. The representations are therefore updated at every step. 
+
+To avoid computing this for each sample, $$s(x)$$ is only computed on the $$M$$ samples with the highest predictive entropy. 
+
+The sample 
+
+
+$$
+x^* = argmax_{x \in D_{oracle}} s(x)
+$$
+
+Is removed from $$D_{oracle}$$ and labeled and added to $$D_{train}$$. 
+
+![](/article/images/MedAL/method.jpg)
+
+## Initial train set construction
+
+Standard active learning techniques label the first training set randomly as they require a trained model for evaluation uncertainty. The authors propose using Oriented FAST and Rotated BRIEF (OBR) descriptors to extract keypoints described by 256-bit vectors for each image. 
+
+The training set construction uses equations 1 and 2 to evaluate the images with the most information. The process is shown in Figure 2.  
```

Comment:
Since you refer to equations by number, you could actually tag the equations with so that they'll be labeled with a number. You can do it by adding `\tag{YOUR_TAG}` at the beginning of the LaTeX code (see [this documentation](https://katex.org/docs/supported.html#annotation) for reference, and [this previous review](https://vitalab.github.io/article/2020/08/14/IsolatingSourcesOfDisentanglementInVAEs.html) for an applied example).